protocol/chainlib/grpc.go

```diff
@@ -69,19 +70,33 @@ func (apip *GrpcChainParser) ParseMsg(url string, data []byte, connectionType st
 		return nil, utils.LavaFormatError("failed to getSupportedApi gRPC", err)
 	}
 
+	// Extract default block parser
+	blockParser := serviceApi.BlockParsing
+
 	apiInterface := GetApiInterfaceFromServiceApi(serviceApi, connectionType)
 	if apiInterface == nil {
 		return nil, fmt.Errorf("could not find the interface %s in the service %s", connectionType, serviceApi.Name)
 	}
 
+	// Check if custom block parser exists in the api interface
+	// Use custom block parser only for URI calls
```

Comment:
we don't have URI in grpc, no need to overwrite block parsing

Reply:
fixed: 524cb1833ba85f59885c6f6e829b7f9d6d08e451